src/main/scala/de/heikoseeberger/sbtheader/HeaderPlugin.scala

```diff
@@ -96,14 +97,17 @@ object HeaderPlugin extends AutoPlugin {
     unmanagedSources in createHeaders := unmanagedSources.value,
     unmanagedResources in createHeaders := unmanagedResources.value,
     createHeaders := createHeadersTask(
-      (unmanagedSources in createHeaders).value.toList ++ (unmanagedResources in createHeaders).value.toList,
+      FileFilter(excludes.value).filter(
+        (unmanagedSources in createHeaders).value.toList ++ (unmanagedResources in createHeaders).value.toList
```

Comment:
Changing this causes the build to fail with:

```
[error] /Users/bene/workspace/projects/sbt-header/src/main/scala/de/heikoseeberger/sbtheader/HeaderPlugin.scala:101: type mismatch;
[error]  found   : Seq[java.io.File]
[error]  required: de.heikoseeberger.sbtheader.Seq[java.io.File]
[error]     (which expands to)  scala.collection.immutable.Seq[java.io.File]
[error]         (unmanagedSources in createHeaders).value ++ (unmanagedResources in createHeaders).value
[error]                                                   ^
[error] one error found
[error] /Users/bene/workspace/projects/sbt-header/src/main/scala/de/heikoseeberger/sbtheader/HeaderPlugin.scala:101: type mismatch;
[error]  found   : Seq[java.io.File]
[error]  required: de.heikoseeberger.sbtheader.Seq[java.io.File]
[error]     (which expands to)  scala.collection.immutable.Seq[java.io.File]
[error]         (unmanagedSources in createHeaders).value ++ (unmanagedResources in createHeaders).value
[error]                                                   ^
[info] No documentation generated with unsucessful compiler run
[error] one error found
[error] (compile:doc) Scaladoc generation failed
[error] (compile:compileIncremental) Compilation failed
[error] Total time: 4 s, completed 16.08.2016 19:15:47
```

This the code has been this way all the time I'd suggest we leave it like this for now.